client/components/TestPlanVersionsPage/index.jsx

```diff
@@ -273,10 +273,48 @@ const TestPlanVersionsPage = () => {
                                                     </PhasePill>
                                                 );
 
+                                                const draftPill = (
+                                                    <PhasePill
+                                                        fullWidth={false}
+                                                    >
+                                                        DRAFT
+                                                    </PhasePill>
+                                                );
+
+                                                if (
+                                                    derivedDeprecatedAtPhase ===
+                                                    'RD'
+                                                ) {
+                                                    return (
+                                                        <>
+                                                            {deprecatedPill}
+                                                            before&nbsp;
+                                                            {draftPill}
+                                                            review
+                                                        </>
+                                                    );
+                                                }
+
+                                                if (
+                                                    derivedDeprecatedAtPhase ===
+                                                    'RECOMMENDED'
+                                                ) {
+                                                    return (
+                                                        <>
+                                                            {deprecatedPill}
```

Comment:
Like above, NVDA returns the speech as `Latest Phase  column 2  Deprecatedafter being approved as Recommended`. The same doesn't happen with VO